app.py

```
@@ -102,127 +106,101 @@ def listt():
-                (message_id, gift_type, count, target_user_id, True, api_admin_name),
+                (
+                    message_id,
+                    gift_type,
+                    gift_amount,
+                    user_name,
+                    True,
+                    api_admin_name,
+                ),
             )
             gift_type = "point" if gift_type == "電電點" else "ticket"
             query = f"update user set {gift_type}={gift_type}+%s where uid=%s"
```

Ruff

Possible SQL injection vector through string-based query construction

(S608)